src/main/java/mekanism/common/registries/MekanismModules.java

```diff
@@ -54,19 +55,18 @@ private MekanismModules() {
           ModuleAttackAmplificationUnit::new, () -> MekanismItems.MODULE_ATTACK_AMPLIFICATION.asItem(), builder -> builder.maxStackSize(4).rarity(Rarity.UNCOMMON)
                 .rendersHUD());
     public static final ModuleRegistryObject<ModuleFarmingUnit> FARMING_UNIT = MODULES.register("farming_unit", ModuleFarmingUnit::new,
-          () -> MekanismItems.MODULE_FARMING.asItem(), builder -> builder.maxStackSize(4).rarity(Rarity.UNCOMMON).exclusive());
-    //TODO: Eventually we may want to come up with a better exclusive method given realistically the shearing unit and farming unit don't need to be
-    // exclusive of each other, but they both should be exclusive in regards to the teleportation unit
+          () -> MekanismItems.MODULE_FARMING.asItem(), builder -> builder.maxStackSize(4).rarity(Rarity.UNCOMMON).exclusive(ExclusiveFlag.INTERACT_BLOCK));
+    // While you can shear blocks, in general they don't overlap with anything that supports the farming unit right click.
```

Comment:
This comment should go away now that we are actually marking it as exclusive for blocks as well.